services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java

```diff
@@ -184,12 +184,15 @@ public Response keycloakInitiatedBrowserLogout(KeycloakSession session, UserSess
             try {
                 SAML2LogoutRequestBuilder logoutBuilder = buildLogoutRequest(userSession, uriInfo, realm, singleLogoutServiceUrl);
                 JaxrsSAML2BindingBuilder binding = buildLogoutBinding(session, userSession, realm);
-                return binding.postBinding(logoutBuilder.buildDocument()).request(singleLogoutServiceUrl);
```

Comment:
This is where the change is not fully backwards compatible. If the user had selected REDIRECT, and depended on POST being used, this will change.